HPXMLtoOpenStudio/measure.rb

```diff
@@ -525,7 +525,9 @@ def create_unit_model(hpxml, hpxml_bldg, runner, model, epw_path, weather, debug
     add_num_occupants(model, runner, spaces)
 
     # HVAC
-    @hvac_unavailable_periods = Schedule.get_unavailable_periods(runner, SchedulesFile::Columns[:HVAC].name, @hpxml_header.unavailable_periods)
+    @hvac_unavailable_periods = []
+    @hvac_unavailable_periods << Schedule.get_unavailable_periods(runner, SchedulesFile::Columns[:SpaceHeating].name, @hpxml_header.unavailable_periods)
+    @hvac_unavailable_periods << Schedule.get_unavailable_periods(runner, SchedulesFile::Columns[:SpaceCooling].name, @hpxml_header.unavailable_periods)
```

Comment:
Why not just create `heating_unavailable_periods` and `cooling_unavailable_periods`? An array feels overkill. And then you don't have to `flatten` below.